objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj

```diff
@@ -826,7 +828,7 @@
 				GCC_WARN_UNUSED_PARAMETER = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				GENERATE_PROFILING_CODE = NO;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Yea, lets close this PR then without landing things and just keep tabs on the betas to see where things end up.